AssetInformationApi/V1/Controllers/AssetInformationApiController.cs

```diff
@@ -134,6 +134,32 @@ public async Task<IActionResult> PatchAsset([FromRoute] EditAssetByIdRequest que
             }
         }
 
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpPatch]
+        [Route("{id}/address")]
+        [LogCall(LogLevel.Information)]
+        public async Task<IActionResult> PatchAssetAddress([FromRoute] EditAssetByIdRequest query, [FromBody] EditAssetRequest asset)
```

Comment:
The second parameter should be `EditAssetAddressRequest`.

Reply:
Good spot - fixed